test/unit/specs/http_connector.js

```diff
@@ -444,7 +444,10 @@ describe('Http Connector', function () {
       var con = new HttpConnection(new Host());
       var respBody = 'i should not be encoded';
       var server = nock('http://localhost:9200')
-      .matchHeader('accept-encoding', undefined)
+      .matchHeader('accept-encoding', function (v) {
```

Comment:
`nock@9` doesn't seem to do a match on `undefined` - changing to a matcher function.